stream/src/js/stream.js

```diff
@@ -2,3 +2,10 @@ $(document).ready(function() {
     actionkit.forms.contextRoot = 'https://go.peoplepower.org/context/';
     actionkit.forms.initForm('act');
 });
+
+$(window).load(function() {
```

Comment:
document.ready fires when dom is loaded while window.load waits for images and other resources.

The answer to your implied question is that we can and should dedup as it doesn't matter in this case.